### ISSUE TITLE
Downloads

### ISSUE THREAD

**Issue by ViralBShah** [JuliaCN/juliacn.github.io#2]
I think it may be a good idea to point downloads directly to the English julialang downloads link. Alternately, the downloads will need to be kept updated.

**Comment by ViralBShah** [comment]
For example, we are now on 1.1.

**Comment by findmyway** [comment]
@Roger-luo @inkydragon @Gnimuc @sunoru
Any comments?

The download link is pointed to USTC currently.
And I create a issue here https://github.com/sunoru/julia-mirror/issues/2 to request updating the mirror.

To avoid constantly updating the download page. Maybe just direct users to the download page (en/ch) would be enough?

**Comment by Gnimuc** [comment]
My two cents are put the official Julia downloads link at the top and remove words like "Current stable release"(当前稳定版本) in the mirror tables(the version number is enough to guide users and there is nothing wrong even the mirror is not updated in time).

**Comment by inkydragon** [comment]
download from `fastly` is slow. (github also use `fastly`, and it's slow too).
So, I think we still need mirrors. 

Maybe we should update USTC's mirror first, and figure out why ZJU's mirror [is down](https://github.com/sunoru/PkgMirrors.jl/issues/7), then set up an another mirrors at THU( https://tuna.moe )

----

About download page. Manual Update is not a good idea.
I recommend that 
- only keep `Long-term support release` (which point to mirrors) 
- point `Current stable release` to English julialang download page, 
- and list mirrors' link. 

Then, only when a new version of `Long-term support release` comes out, will we update download page.

**Comment by ViralBShah** [comment]
Maybe provide both: regular English download links and fast mirror?

We may want some tooling to auto generate the downloads then.

**Comment by ViralBShah** [comment]
Cc @staticfloat in case he has thoughts on this.

**Comment by sunoru** [comment]
The 502 error on ZJU mirror has been fixed, and the release information has been updated so Julia v1.1 can now be downloaded from `/releases/v1.1` on the mirrors. (For example, https://mirrors.zju.edu.cn/julia/releases/v1.1/ )

I did use some scripts to auto generate the downloads: https://github.com/sunoru/julia-mirror/blob/master/scripts/make_releaseinfo.py

It is just by parsing the download page on the official website. Hope there is a more stable way to get the release information.

I forgot to set up an automatic run, and I didn't run it for long, so v1.1 was not available on the mirrors in time. Maybe I should write some scripts to do so and automatically update the download page of JuliaCN.

After this is done I think the download links for all the versions will be stable and we only need the ones from the mirrors.
